javascript/app.js

```diff
@@ -0,0 +1,905 @@
+// Declaring/Assigning is usually referred to as Defining
+
+// How do we change the value of a variable?
+// We change a variable's value by reassigning one.
+// However one must pay attention to the declared variable type!
+// const variables cannot be reassigned!
+// let ex1 = 5;
+// const ex2 = 7;
+// ex1 = ex1*2;
+// ex2 = ex2*5; //==> app.js:20 Uncaught TypeError: Assignment to constant variable.
+
+// How do we assign an existing variable to a new variable?
+// A new empty variable can be assigned any object, so another variable is cool!
+// const variable = 5;
+// const newVar = variable;
+// console.log(newVar); //==> 5
+
+// Remind me, what are declare, assign, and define?
+// Declare: Bring variable into existence
+// Assign: assign it a value using =
+// Define: Declare+Assign
+
+// What is pseudocoding and why should you do it?
+// Pseudocode: Writing out in plain text the processes that will make up the code
+// Awesome for being able to take a step back and keep track of the mechanics/where we are
+// Awesome for breaking down into tiny bits the mechanics of the code
+
+// What percentage of time should be spent thinking about how you're going to solve a problem vs actually typing in code to solve it?
+// The Masters once said: "Timewise, programming is 75-90% planning, and 10-25% actually typing code."
+
+// 🔴 Commit.
+////////////////////////////////////////
+
+
+// B. Strings
+
+// Create a variable called firstVariable.
+//==> Must declare variable with a let because we will be changing its value
+let firstVariable;
+// Assign it the value of the string "Hello World"
+firstVariable = "Hello World";
+// Change the value of this variable to some number.
+firstVariable = 9;
+// Store the value of firstVariable in a new variable called secondVariable
+let secondVariable = firstVariable;
+// Change the value of secondVariable to any string.
+secondVariable = "Goodbye World";
+// What is the value of firstVariable?
+console.log(firstVariable); // ==> 9
+// Create a variable called yourName and set it equal to your name as a string. 
+let yourName = "Charlotte";
+// Then, write an expression that takes the string "Hello, my name is " and the variable yourName so that it returns a new string with them concatenated.
+// ex: Hello, my name is Jean Valjean
+console.log("Hello, my name is " + yourName);;
+
+// 🔴 Commit.
+////////////////////////////////////////
+
+
+// C. Booleans
+
+// Using the provided variable definitions, replace the blanks so that all log statements print true in the console. Answers should be all be valid JS syntax and not weird things that don't make sense but happen to print true to the console.
+  const a = 4;
+  const b = 53;
+  const c = 57;
+  const d = 16;
+  const e = 'Kevin';
+
+  console.log(a <= b);
+  console.log(c >= d);
+  console.log('Name' === 'Name');
+  console.log(true !== false);
+  console.log(false === false === false === false === false === false === true);
```

Comment:
this syntax is meaningless. desired result was `false || false || ..... || true`